src/command/upload.ts

```diff
@@ -31,6 +33,8 @@ export class Upload extends RootCommand implements LeafCommand {
   @Option({ key: 'tag-polling-trials', describe: 'After the given trials the tag polling will stop', default: 15 })
   public tagPollingTrials!: number
 
+  public uploadAsFileList = false
```

Comment:
It is only intended to be used internally, set from another command for example.

It is a workaround for uploading single files as a single-element browser-like File arrays.

I would not spend too much time fleshing this out, since this will be removed ASAP.